### ISSUE TITLE
replace all the xpra logos

### ISSUE THREAD

**Issue by totaam** [Xpra-org/xpra#1342]
The ones we have were made by tweaking the [X.org](https://www.x.org/wiki/) one, we should use our own one.

Here are the main ones we need:
* the background shown in the macosx PKG installer
* MacOS icons: will need 512x512 and 512x512@x2 for [Optimizing for High Resolution](https://developer.apple.com/library/content/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Optimizing/Optimizing.html)
* `xpra.png` the main one, shown in various places
* `xpra-logo.bmp` the MS Windows installer
* `xpra.bmp` MS Windows installer small version
* `xpra.ico` MS Windows ICO version, which needs to have various sizes for the system tray - MS Windows ICO version, which needs to have various sizes for the system tray, as low as 16x16, see [What is the size of the icons in the system tray?](http://stackoverflow.com/questions/3531232/) - (the smaller versions may look substantially different)
* `xpra_txt.ico` icon for text commands (maybe just clean it up a bit?)
* `favicon.png` web shortcut icon (forgot to change the colour on this one!)
* `favicon.ico` web shortcut icon in ICO format
* `xpra-logo.png` shown in the [HTML5 client](http://xpra.org/html5/connect.html)

**Comment by totaam** [comment]
This is going to become more relevant with #387, so I asked Claude AI to help and it came up with this SVG:
![Image](https://github.com/user-attachments/assets/adb8843c-c490-4a22-af94-9b43dc27965c)

And this is what it looks like at 48x48:
<img width="48" height="48" alt="Image" src="https://github.com/user-attachments/assets/13c05198-9971-4c9a-bcf8-7ac51f585177" />


---


Alternative that moves the X to the background:
![Image](https://github.com/user-attachments/assets/53574232-f1af-4250-a6ac-94468a624204)

**Comment by amirlatifi** [comment]
I have some ideas:
- Keeping a single X might be enough.
- Using two Xs, or a single X in two colors, could convey the idea of mirroring.
- The concept of mirroring can be applied to the entire icon, not just the X.

<img width="1024" height="1024" alt="Image" src="https://github.com/user-attachments/assets/e9421ac0-cfcd-4829-b8f7-e3365204c0e3" />

**Comment by totaam** [comment]
@amirlatifi I've merged your icon for testing, mostly to see the reaction and to get a feel for what other changes would be required.

**Comment by allo-** [comment]
If you'd like some feedback:
- I was searching, if my Debian testing got an Xorg update with a new logo or maybe updated an icon theme: So indeed the old logo may have looked a bit too much like Xorg.
- In my panel it is the only icon that looks like it would have a frame and one needs to look twice to see that it's part of the icon. The icon may look a bit weird if a panel adds an own frame.
  - Maybe showing a window with background color would help, similar to the KDE systemsettings icon (leftmost in the screenshots).
- The old icon worked better when put into the lower right corner of another icon. Maybe use a version without the border when putting it on top of another icon?
- Overall: Not bad, but maybe a bit generic. I like it much better than the first drafts in this thread.

That's how it looks in a KDE panel:

<img width="244" height="46" alt="Image" src="https://github.com/user-attachments/assets/bcc17afe-e95e-4468-90e2-3a66ff9a164e" />

<img width="244" height="46" alt="Image" src="https://github.com/user-attachments/assets/b2e5c67a-4901-400f-8cd7-cf2d6f3e0429" />

**Comment by totaam** [comment]
@allo- how about 49c67ade6c9f87e1a15306c7eb7ea29dda78d158 ?

**Comment by allo-** [comment]
This could work better, I can try it later.

I wonder about dark UI themes. In the 48x48 Preview on Github I barely see the very light parts, on a dark panel one might not see much of the X. Or one possibly provides two versions or a monochrome version that can be used in black or white.

Also the large Icon has the left circle slightly cut off at top. Is this maybe some viewbox issue in an SVG file?

**Comment by totaam** [comment]
> Or one possibly provides two versions or a monochrome version that can be used in black or white

Themes are a PITA, very hard to know what is in use or how to accommodate it.

> Also the large Icon has the left circle slightly cut off at top

Will fix!

**Comment by allo-** [comment]
> Themes are a PITA, very hard to know what is in use or how to accommodate it.

Indeed. But I think some distributions use monochrome icon schemes to be a bit more flexible and supporting that is good, in particular for the tray icon. Many other custom themes should create an own icon if needed.

To the new icons:
- Something is caching the icon for the badge, restarting plasmashell did not help. I'm not sure how to get it to reload it but maybe I can get it to work later.
- I think the dot now looks wrong. Previously it were implied to be a window button, now it's unclear what it should mean
- The light dashes look like they might be rendering errors
- The dot looks like a rendering problem or dirt on the screen in the tray icon. The light dash is barely visible
- The overall shape is fine and I suppose it would work as a badge.

Panel: <img width="58" height="48" alt="Image" src="https://github.com/user-attachments/assets/b228227a-7a29-4416-bf50-b03a045f3cff" />

Tray: <img width="31" height="23" alt="Image" src="https://github.com/user-attachments/assets/791ef281-0ea7-4ea3-a6be-c4322b6db657" />

**Comment by totaam** [comment]
Not heard any major complaints.
